contracts/PublicResolver.sol

```diff
@@ -0,0 +1,297 @@
+pragma solidity ^0.8.0;
+
+import './ENS.sol';
+
+/**
+ * A simple resolver anyone can use; only allows the owner of a node to set its
+ * address.
+ */
+contract PublicResolver {
+  bytes4 constant INTERFACE_META_ID = 0x01ffc9a7;
+  bytes4 constant ADDR_INTERFACE_ID = 0x3b3b57de;
+  bytes4 constant CONTENT_INTERFACE_ID = 0xd8389dc5;
+  bytes4 constant NAME_INTERFACE_ID = 0x691f3431;
+  bytes4 constant ABI_INTERFACE_ID = 0x2203ab56;
```

Comment:
as I see below, this implementation is commented out, so either take out this interface ID or implement it as https://docs.ens.domains/ens-improvement-proposals/ensip-4-support-for-contract-abis states.